js/testapps/cat-eval/src/pdf_rag.ts

```diff
@@ -67,26 +68,6 @@ export const pdfQA = defineFlow(
     const llmResponse = await generate({
       model: geminiPro,
       prompt: augmentedPrompt,
-      config: {
-        safetySettings: [
-          {
-            category: 'HARM_CATEGORY_HATE_SPEECH',
-            threshold: 'BLOCK_NONE',
-          },
-          {
-            category: 'HARM_CATEGORY_DANGEROUS_CONTENT',
-            threshold: 'BLOCK_NONE',
-          },
-          {
-            category: 'HARM_CATEGORY_HARASSMENT',
-            threshold: 'BLOCK_NONE',
-          },
-          {
-            category: 'HARM_CATEGORY_SEXUALLY_EXPLICIT',
-            threshold: 'BLOCK_NONE',
-          },
-        ],
-      },
```

Comment:
Calling it out *just* in case - you might still want this (evaluators should probably never be allowed to get blocked for content).

Reply:
This setting is for the flow under test, not the evaluators.